src/app-layer-ftp.c

```diff
@@ -189,6 +189,8 @@ static int FTPParseRequestCommand(void *ftp_state, uint8_t *input,
     if (input_len >= 4) {
         if (SCMemcmpLowercase("port", input, 4) == 0) {
             fstate->command = FTP_COMMAND_PORT;
+        } else if (SCMemcmpLowercase("auth tls", input, 8) == 0) {
```

Comment:
If input_len is bigger than sizeof("auth tls") then we will read after the limit. Not sure we want that.